examples/css/timetree_chrome_devtools.css

```diff
@@ -1,60 +0,0 @@
-.chrome-devtools-timetree, .chrome-devtools-timetree .label text {
```

Comment:
tl;dr Separating it out just for the sake of is unnecessary.

I think if we're going to make the chrome theme the new default, no need to do extra work as a stopgap that will just be immediately thrown away. That said, I'm leaving the style changes to you since you know the overrides.